docroot/sites/all/modules/custom/midcamp_eventbrite/includes/AttendeeProcessor.php

```diff
@@ -0,0 +1,161 @@
+
+  protected function fetchAttendees() {
+    $response = EventbriteApi::request('GET', EVENTBRITE_API_ENDPOINT . '/events/' . EventbriteApi::eventId() . '/attendees');
+    $this->attendees += $response['attendees'];
+    $params = array('id' => variable_get('midcamp_eventbrite_event_id'));
+    if (isset($response['pagination']) && $response['pagination']['page_count'] > 1) {
+      $current_page = $response['pagination']['page_number'];
+      while ($current_page <= $response['pagination']['page_count']) {
+        $current_page++;
+        $paginated_response = EventbriteApi::request('GET', EVENTBRITE_API_ENDPOINT . '/events/' . EventbriteApi::eventId() . '/attendees', array(
+          'page' => $current_page,
+        ));
+        $this->attendees += $paginated_response['attendees'];
+      }
+    }
+  }
+
+  public function getAttendees() {
+    return $this->attendees;
+  }
+
+  /**
+   * Loads attendee.
+   * @param array $attendee
+   * @return bool|mixed
+   * @throws \Exception
+   */
+  public function loadUserByAttendee(array $attendee) {
+    $subscribe = TRUE;
+    if ($user = midcamp_eventbrite_user_load_by_order_id($attendee['order_id'])) {
+      // Verify they are subscribed to mailchimp. Resistance is futile.
+      $aw = entity_metadata_wrapper('user', $user);
+      $subscribed_field = $aw->field_subscribe->value();
+      $subscribe = ($subscribed_field && array_key_exists('subscribe', $subscribed_field) && $subscribed_field['subscribe'] == FALSE);
+    }
+    elseif ($user = user_load_by_mail($attendee['profile']['email'])) {
+      // Then look for existing user account for email.
+      $user->order_id = $attendee['order_id'];
+      user_save($user);
+      _user_mail_notify('register_eventbrite_existing', $user);
+    }
+    else {
+      // Last resort we'll create a new user.
+      $user = midcamp_eventbrite_user_create($attendee);
+      $user->order_id = $attendee['order_id'];
+      $edit['pass'] = $user->password;
+      user_save($user, $edit);
+      _user_mail_notify('register_eventbrite', $user);
+    }
+
+    if ($subscribe === TRUE) {
+      midcamp_eventbrite_mailchimp_subscribe('user', $user, 'field_subscribe');
+    }
+
+    return $user;
+  }
+
+  /**
+   * Creates profile for account and populates with answers from Eventbrite.
+   * @param \stdClass $user
+   * @param array $attendee
+   * @return \Profile
+   */
+  public function conferenceProfileSync(stdClass $user, array $attendee) {
+    $type_name = 'conference_details';
+    $p = profile2_load_by_user($user, $type_name);
+    if (!$p) {
+      $p = profile2_create(array('type' => $type_name));
+      $p->setUser($user);
+      $p->is_new = TRUE;
+    }
+
+    if (!empty($attendee['answers'])) {
+      $pw = entity_metadata_wrapper('profile2', $p);
+      foreach ($attendee['answers'] as $answer) {
+        switch ($answer->question_id) {
+          // I would be interested in attending a Friday evening social event.
+          case 9340079:
```

Comment:
I'm pretty sure this will be different. Even if the questions end up being the same, the $answer ids will be different. While it was nice to have this info available to the Drupals, it is easily attainable from Eventbrite. Just something to consider if changing the profile2 to match the new EB questions proves to be too time-consuming.